dashboard/README.md

```diff
@@ -0,0 +1,45 @@
+- To run dashboard, one needs to create a ~/.ssh/config file similar to the following and keep the sockets to the 4 hosts open.
+- By default, dashboard will try log into maui01/mahuika01.
+- After 3 consecutive unsuccessful ssh commands, dashboard will try log into maui02/mahuika02.
+- After 6 consecutive unsuccessful ssh commands, dashboard will exit with error message.
+```
+Host mahuika
+   User melody.zhu
+   #HostName login.mahuika.nesi.org.nz
```

Comment:
Remove those comments for the readme.

Could be a good idea to leave mahuika as the default and then have mahuika01 point to 01.